Odata-docs/webapi-8/fundamentals/query-options.md

```diff
@@ -0,0 +1,547 @@
+---
+title: "OData Query Options in ASP.NET Core OData 8"
+description: "This documentation describes how to use the supported query options."
+
+author: KenitoInc
+ms.author: kemunga
+
+ms.date: 12/16/2022
+---
+# OData Query Options in ASP.NET Core OData 8
+**Applies To**:[!INCLUDE[appliesto-webapi](../../includes/appliesto-webapi-v8.md)]
+
+AspNetCore OData 8.x provides out of the box querying capabilities.
+
+A query option is basically requesting that a service perform a set of transformations such as filtering, sorting, etc. to its data before returning the results. 
+
+There are various types of query options.
+- `System Query Options` - Query options that are defined by OData. System query options are prefixed with the dollar ($) character, which is optional in OData 4.01. Example is `$filter`.
+- `Parameter aliases` - Parameter aliases can be used in place of literal values in entity keys, function parameters, or within a $compute, $filter or $orderby expression. Example `http://host/service.svc/Employees?$filter=Region eq @p1&@p1='WA'`.
+- `Custom Query Options` - Custom query options not defined in the OData specification. They are defined by the users. They MUST NOT begin with the `$` or `@` character and MUST NOT conflict with any OData-defined system query options. Example `http://host/service/Products?debug-mode=true`.
+
+See more information on the types of query options [here](/odata/concepts/queryoptions-overview).
+
+This tutorial demonstrates how to use query options in AspNetCore OData 8.x.
+
+## Model class
+We will define our model as follows:
+
+```csharp
+public class Customer
+{
+    [Key]
+    public int Id { get; set; }
+    public String Name { get; set; }
+    public int Age { get; set; }
+    public List<Order> Orders { get; set; }
+}
+
+public class Order
+{
+    [Key]
+    public int Id { get; set; }
+    public int Price { get; set; }
+    public int Quantity { get; set; }
+}
+```
+
+We have a `Customer` entity with `Orders` navigation property.
```

Comment:
@KenitoInc to a beginner it may be unclear how Orders property on the Customer class maps to OData EDM navigation property concept. 
Consider a brief explanation of the mapping and a link to a Data Modelling doc.